guides/md/keras_cv/classification_with_keras_cv.md

```diff
@@ -496,10 +599,45 @@ keras_cv.visualization.plot_image_gallery(
 )
 ```
 
+<div class="k-default-codeblock">
+```
+WARNING:tensorflow:Using a while_loop for converting RngReadAndSkip cause there is no registered converter for this op.
```

Comment:
Do you know the cause of all these warnings? Is this something we could fix?

Reply:
Yes, its coming from the call - 
```
import numpy as np
import keras_cv

x = np.random.rand(1, 5, 5, 3)
random_cutout = keras_cv.layers.RandomCutout(width_factor=0.4, height_factor=0.4)
# Call of `random_cutout` causes this warning
_ = random_cutout(x)
```
From this line - https://github.com/keras-team/keras-cv/blob/master/keras_cv/utils/fill_utils.py#L84
`images = tf.where(is_rectangle, fill_values, images)`
* `is_rectangle` shape is `(1, 5, 5, 1)` (Bool)
* `fill_values` shape is `(1, 5, 5, 3)` (float32)
* `images` shape is `(1, 5, 5, 3)` (float32)

Not sure why its happening.

Reply:
It doesn't look actionable. Please remove the warnings from the `md` file manually, we'll merge.

Reply:
Thanks. Done